### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by ewelina-jazurek**
creating files for practice css exercices

### CODE REVIEW

File: conflicts.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+<body>
+    <header class="navigation">
+        <p class="blurb">Lorem ipsum dolor sit amet, consectetur adipiscing elit. In mollis dui vitae varius cursus.
+            Donec sit amet ex
+            nec
+            elit tempus ultricies. Morbi sed felis libero. Praesent euismod odio vitae lectus dignissim ullamcorper.
+            Curabitur tincidunt, urna vitae pretium egestas, velit nibh blandit nulla, venenatis accumsan mauris sapien
+            vitae sapien. Vivamus id lacus ut eros sodales gravida. Aenean a blandit magna. Mauris congue nibh nec metus
+            faucibus cursus. Nunc et vulputate tellus, eu condimentum lectus.</p>
```
Comment (grzegorz-jazurek) [inline conflicts.html:32]: it's better to have opening and closing tags in new lines like:
```
<p>
  lorem ipsum
</p>
```

File: floating.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <style>
+        div {
+            background-color: cornflowerblue;
+        }
+
+        p {
+            width: 50px;
+            height: 50px;
+            border: 10px darksalmon;
+
```
Comment (grzegorz-jazurek) [inline floating.html:16]: not needed new line

File: floating.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        }
+
+        #p1 {
+            background-color: deeppink;
+            float: left;
+            height: 150px;
+
```
Comment (grzegorz-jazurek) [inline floating.html:23]: remove empty line

File: font-size.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    <title>font size</title>
+    <style>
+        * {
+            box-sizing: border-box;
+            margin: 0;
+            padding: 0;
+        }
+
+        body {
+
```
Comment (grzegorz-jazurek) [inline font-size.html:15]: not needed empty spaces

**Review by grzegorz-jazurek — CHANGES_REQUESTED** [review]
please fix formatting and white spaces

**Comment by ewelina-jazurek** [comment]
corrections made

File: CSS-L1.html [hunk 5]
```diff
@@ -8,8 +8,7 @@
     <title>Anatomy of CSS Rule</title>
     <style>
         p {
-<<<<<<< HEAD
-            color: greenyellow;
+            <<<<<<< HEAD color: greenyellow;
```
Comment (grzegorz-jazurek) [inline CSS-L1.html]: git conflicts

Comment (ewelina-jazurek) [reply CSS-L1.html]: fixed

**Pull request merged**